R/teal_slices.R

```diff
@@ -271,8 +271,9 @@ setdiff_teal_slices <- function(x, y) {
 #'
 coalesce_r <- function(x) {
   checkmate::assert_list(x)
-  if (all(vapply(x, is.atomic, logical(1L)))) {
-    return(Filter(Negate(is.null), x)[[1L]])
+  xnn <- Filter(Negate(is.null), x)
+  if (all(vapply(xnn, is.atomic, logical(1L)))) {
+    return(xnn[[1L]])
```

Comment:
This is already in on `main`, I worry there are some merging artefacts on this branch.

Reply:
branch is up to date with main. let try with changing base branch.

Reply:
its not there anymore.